client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -181,7 +181,7 @@ class CheckoutThankYouHeader extends PureComponent {
 
 			return translate(
 				'Your domain {{strong}}%(domain)s{{/strong}} was added to your site, but ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Your domain {{strong}}%(domain)s{{/strong}} was added to your site, but but the transfer process can take up to 5 days to complete.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).